projects/playground/src/app/app-routing.module.ts

```diff
@@ -3,17 +3,33 @@ import { Routes, RouterModule } from '@angular/router';
 import { ProtectedComponent } from './protected/protected.component';
 import { AuthGuard } from 'projects/auth0-angular/src/lib/auth.guard';
 import { UnprotectedComponent } from './unprotected/unprotected.component';
+import { ChildRouteComponent } from './child-route/child-route.component';
+import { NestedChildRouteComponent } from './child-route/nested-child-route.component';
 
 const routes: Routes = [
+  {
+    path: '',
+    component: UnprotectedComponent,
+    pathMatch: 'full',
+  },
+  {
+    path: 'child',
+    component: ChildRouteComponent,
+    canActivateChild: [AuthGuard],
+    children: [{ path: 'nested', component: NestedChildRouteComponent }],
+  },
   {
     path: 'protected',
     component: ProtectedComponent,
     canActivate: [AuthGuard],
   },
   {
-    path: '',
-    component: UnprotectedComponent,
-    pathMatch: 'full',
+    path: 'lazy',
+    canLoad: [AuthGuard],
+    loadChildren: () =>
+      import('./lazy-module/lazy-module.module').then(
```

Comment:
why is this different syntax needed? can't we just do something like `component: LazyModuleComponent`?